### DISCUSSION TITLE
Why removing readingTime?

### DISCUSSION THREAD

**Discussion by tomiRohi** [arthelokyo/astrowind#106] [Q&A]
Hello everybody, I'm using this template to build blog.rohisommiers.com

I love the reading Time remark plugin and I saw that it was removed from project a few days ago. This was for performance reasons, incompatibility, or maybe something else?

Thank you in advance.

**Comment by rhizodigital** [comment]
Yes I wondered this, Im having problems implementing reading time in my own template. Im only having issue with content collection and remarkPluginFrontmater just returning undefined.

**Comment by prototypa** [comment]
Hi @tomiRohi 

When we migrated **AstroWind** to _Astro 2.0_ it was not our goal to remove _Reading Time_.

However, as @rhizodigital mentions, the system returned `undefined` all the time and we did several tests. It occurred to us that perhaps we could have temporarily added the functionality inside the collection's `transform` function, or in the utility that does the normalization but it seemed a bit forced and we decided to temporarily remove it until we found the problem.

Today we will be checking again, if we get it to work we will add it. If anyone has had the problem and found a solution please let us know. Thanks.

----
**UPDATE**: I have opened an issue in the official astro repo.: https://github.com/withastro/astro/issues/6016

----
**UPDATE**: I didn't realize there was another issue about this. As I see it has already been solved and the fixes will come out in some soon **Astro** release. https://github.com/withastro/astro/issues/5859

As soon as that **Astro** release is ready, the reading time will appear again in **AstroWind**.

**Comment by prototypa** [comment]
Hi everyone.

With the [Astro 2.0.4 version](https://github.com/withastro/astro/releases/tag/astro%402.0.4), the data from the remark plugins is already received. 

The `readingTime` plugin in **AstroWind** is now active.
